### PR TITLE
Stop sending an extra parameter to ToFedora::Identity

### LINKED ISSUE #2642: Unable to update an APO: ArgumentError.
```dor-services-app_1  | F, [2021-04-02T19:26:21.009456 #1] FATAL -- : [0603bb0f-e8f8-4894-b9f9-a48f8c255f96] ArgumentError (unknown keyword: :object_type):
dor-services-app_1  | F, [2021-04-02T19:26:21.009666 #1] FATAL -- : [0603bb0f-e8f8-4894-b9f9-a48f8c255f96]
dor-services-app_1  | F, [2021-04-02T19:26:21.010034 #1] FATAL -- : [0603bb0f-e8f8-4894-b9f9-a48f8c255f96] app/services/cocina/to_fedora/identity.rb:9:in `apply'
dor-services-app_1  | [0603bb0f-e8f8-4894-b9f9-a48f8c255f96] app/services/cocina/object_updater.rb:76:in `update_apo'
dor-services-app_1  | [0603bb0f-e8f8-4894-b9f9-a48f8c255f96] app/services/cocina/object_updater.rb:36:in `run'
dor-services-app_1  | [0603bb0f-e8f8-4894-b9f9-a48f8c255f96] app/services/cocina/object_updater.rb:16:in `run'
dor-services-app_1  | [0603bb0f-e8f8-4894-b9f9-a48f8c255f96] app/controllers/objects_controller.rb:41:in `update'
```

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

Fixes #2642

### CODE REVIEW

File: spec/requests/update_metadata_spec.rb [hunk 1]
```diff
@@ -956,7 +956,7 @@
   context 'when an APO is provided' do
     let(:item) do
       Dor::AdminPolicyObject.new(pid: druid,
-                                 label: label).tap do |item|
+                                 label: 'old value').tap do |item|
```
Comment (peetucket) [inline spec/requests/update_metadata_spec.rb:959]: why this change?

Comment (jcoyne) [reply spec/requests/update_metadata_spec.rb:959]: This triggers the bug.  Previously there was a check added that said "don't run this code unless the label changed", this causes the label to appear to have changed.

**Review by ndushay — APPROVED** [review]
I'm approving THIS PR to fix the production problem, but I would like to see the infrastructure integration test for creating an APO ... also UPDATE the APO so we don't break this in the future.

**Comment by jcoyne** [comment]
@Naomi Dushay the tests that exposed this are in the Argo test suite, so it it necessary to duplicate that and make the integration tests slower?

File: spec/requests/update_metadata_spec.rb (hunk 1 above)
Comment (peetucket) [reply spec/requests/update_metadata_spec.rb:959]: ahh
